.github/workflows/manage-stale-issue-pr.yml

```diff
@@ -0,0 +1,40 @@
+name: Manage Stale Issues and PRs
+
+on:
+  schedule:
+    # Run once every day at 9 AM UTC
+    - cron: 00 9 * * *
+
+jobs:
+  stale-issues-and-prs:
+    name: Comment on possible stable issues and PRs, and close stale PRs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          operations-per-run: 50  # Max number of operations per run (including fetch or close issues and PRs, set or update labels, add comments, delete branches, etc.)
+          start-date: "2024-05-05"  # Skip stale action for issues/PRs created before 5th May 2024
```

Comment:
If we merge this PR now, the action will ignore all issues and PRs that are created before 05/05/2024, which is almost all of them.

Should we manage the current backlog first, remove this option, then merge the PR?
cc: @jrobinAV @FabienLelaquais

Reply:
Let's review them during the sprint meeting.

Reply:
All the pending issues have been reviewed. I added a comment in case of no activity for more than 2 weeks.

Reply:
```suggestion
```
This can be removed now